src/main.cpp

```diff
@@ -307,7 +313,7 @@ std::string    msNeuralResponse;
 std::string    msHDDSerial;
 //When syncing, we grandfather block rejection rules up to this block, as rules became stricter over time and fields changed
 
-int nGrandfather = 1035000;
+int nGrandfather = 1034700;
```

Comment:
I reverted this to previous value, but even then I feel this is too high. Should be below v8.